docs/pages/get-started/installation.mdx

```diff
@@ -25,14 +26,19 @@ Linux, macOS and Windows are all supported as your development machine. You can
 To use Expo CLI, you need to have the following tools installed on your developer machine:
 
 - [Node.js LTS release](https://nodejs.org/en/) - Only Node.js LTS releases (even-numbered) are recommended.
-  <CALLOUT theme="secondary">As Node.js [officially states](https://nodejs.org/en/about/releases/), "Production applications should only use Active LTS or Maintenance LTS releases". You can install Node.js using a version management tool (such as `nvm` or `volta` or any other of your choice) to switch between different Node.js versions.</CALLOUT>
+  <CALLOUT theme="secondary">
+    As Node.js [officially states](https://nodejs.org/en/about/releases/), "Production applications
+    should only use Active LTS or Maintenance LTS releases". You can install Node.js using a version
+    management tool (such as `nvm` or `volta` or any other of your choice) to switch between
+    different Node.js versions.
+  </CALLOUT>
```

Comment:
ATM we need to keep the content of CALLOUT inline, otherwise the styling stops working.

<img width="1138" alt="Screenshot 2023-05-27 at 14 13 53" src="https://github.com/expo/expo/assets/719641/aa2aef32-ef31-4938-9ddd-c25d733b03ad">

Reply:
I've reverted these changes. Sorry this was done by Prettier (I think).